src/MCPServer/share/mysql_dev_9265_index_after_processing_decision.sql

```diff
@@ -0,0 +1,46 @@
+-- Remove the original "index transfer contents" microservice during ingest
+
+SET @updateTransferIndex = 'd46f6af8-bc4e-4369-a808-c0fedb439fef' COLLATE utf8_unicode_ci;
+SET @sendToBacklog = 'abd6d60c-d50f-4660-a189-ac1b34fafe85' COLLATE utf8_unicode_ci;
+
+SET @index_transfer_contents = 'eb52299b-9ae6-4a1f-831e-c7eee0de829f' COLLATE utf8_unicode_ci;
+SET @create_transfer_metadata_xml = 'db99ab43-04d7-44ab-89ec-e09d7bbdc39d' COLLATE utf8_unicode_ci;
+
+-- "Check for specialized processing" should point at the chainlink after indexing
+UPDATE TasksConfigsUnitVariableLinkPull
+    SET defaultMicroServiceChainLink=@create_transfer_metadata_xml
+    WHERE defaultMicroServiceChainLink=@index_transfer_contents;
+
+-- "Identify DSpace METS file" should point at the chainlink after indexing
+UPDATE MicroServiceChainLinks
+    SET defaultNextChainLink=@create_transfer_metadata_xml
+    WHERE pk='8ec0b0c1-79ad-4d22-abcd-8e95fcceabbc';
+UPDATE MicroServiceChainLinksExitCodes
+    SET nextMicroServiceChainLink=@create_transfer_metadata_xml
+    WHERE nextMicroServiceChainLink=@index_transfer_contents;
```

Comment:
Updated; deleted the STC and MSCLExitCodes rows in this file, and the MSCL and TC in the deletion file (since they may not be deletable).